### PR TITLE
[ADD] DDT type multi-company

### PR DESCRIPTION
**Pull request by SimoRubi**
This covers the following use case:

1. User in Company1: Define the ddt.type *DDT_type* and bind that to a sequence *DDT_seq*;
2. User in Company2: Create new delivery order and place it in a new DDT of type *DDT_type*;
3. User in Company2: 'Put in pack'

Behavior before this PR:
```
The requested operation cannot be completed due to security restrictions.
Please contact your system administrator.

(Document type: ir.sequence, Operation: read, Records: 28, User: 10)
```
Behavior after this PR:
Package created

### CODE REVIEW

File: l10n_it_ddt/data/ddt_data.xml [hunk 1]
```diff
@@ -16,6 +16,13 @@
             <field name="sequence_id" ref="seq_ddt" />
         </record>
 
+        <record id="ddt_type_comp_rule" model="ir.rule">
```
Comment (eLBati) [inline l10n_it_ddt/data/ddt_data.xml]: please move this record to a file under ```security``` dir

**Comment by SimoRubi** [comment]
@eLBati done

**Comment by SimoRubi** [comment]
@tafaRU fixed up, and better commit message

**Comment by eLBati** [comment]
@SimoRubi da includere nella PR per 11.0

**Pull request merged**